.travis.yml

```diff
@@ -47,6 +47,13 @@ matrix:
   - if: branch !~ /(^branch-.*-built)/
     language: node_js
     env: WP_TRAVISCI="yarn build"
+  - name: "E2E tests"
+    php: 7.2
+    env: WP_BRANCH=latest RUN_E2E=1
```

Comment:
Should you add `WP_MODE=single` for explicit clarity?

Reply:
definitely, but I doubt that this code will be merged to master anyway.  Instead, we would rely on external site configurations such as JN